packages/convertlit/build.sh

```diff
@@ -0,0 +1,37 @@
+TERMUX_PKG_HOMEPAGE='http://www.convertlit.com/'
```

Comment:
Since this is an indirect dependency it would be good to mention what package requires it.
```suggestion
# Dependency for ebook-tools
TERMUX_PKG_HOMEPAGE='http://www.convertlit.com/'
```